app/code/community/HE/TwoFactorAuth/Model/Observer.php

```diff
@@ -156,7 +156,7 @@ public function check_twofactor_active($observer)
         }
     }
 
-    /* 
+    /*
```

Comment:
Let's not change this indentation for no reason. Force push this out, I don't want it in the `git blame`.